pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
```

Comment:
@Akasurde the explicit `wheel` is redundant. Some of the older docs suggested that it was needed, but it's not. It was only in those suggestions due to historical confusion.
setuptools' PEP 517 back-end has always declared it as a dependency, since it was first implemented.
I removed the suggestion from the setuptools' docs a while ago: https://github.com/pypa/setuptools/commit/f7d30a9529378cf69054b5176249e5457aaf640a.